tasks/install-matlab/v1/task.json

```diff
@@ -27,8 +26,8 @@
             "type": "string",
             "label": "Products",
             "required": false,
-            "defaultValue": "MATLAB Parallel_Computing_Toolbox",
-            "helpMarkDown": "Space-separated list of products to install. If a product name contains white-space characters, replace them with underscores. For the full list of available products and their names, see https://www.mathworks.com/products.html."
+            "defaultValue": "MATLAB",
+            "helpMarkDown": "Products to set up in addition to MATLAB, specified as a list of product names separated by spaces"
```

Comment:
set up --> install

Do you terminate other helpMarkDown elements in this file? If so, a period is needed.

Reply:
updated